index.html

```diff
@@ -260,6 +272,12 @@ <h3 id="active">Active</h3>
           <p>For <dfn>active fingerprinting</dfn>, we also consider techniques where a site runs JavaScript or other code on the local client to observe additional characteristics about the browser, user, device or other context.</p>
           <p>Techniques for active fingerprinting might include accessing the window size, enumerating fonts or plug-ins, evaluating performance characteristics, reading from device sensors, and rendering graphical patterns. Key to this distinction is that <a>active fingerprinting</a> takes place in a way that is potentially detectable on the client.</p>
         </section>
+        <section>
+          <h3 id="ephemeral">Ephemeral</h3>
+          <p><dfn>Ephemeral fingerprinting</dfn> is a technique to associate separate simultaneous sessions on a device with one another using observations of events that occurr near simultaneously on multiple origins [[?EPHEMERAL-FINGERPRINTING]].  These events are typically fired as a result of a change in hardware or environment, such as MediaDevices.devicechange [[?mediacapture-streams]].</p>
```

Comment:
(I suggest wrapping lines so that proposed edits like this can be clearer.)

```suggestion
          <p><dfn>Ephemeral fingerprinting</dfn> is a technique to associate separate simultaneous sessions on a device with one another using observations of events that occur near simultaneously on multiple origins [[?EPHEMERAL-FINGERPRINTING]].  These events are typically fired as a result of a change in hardware or environment, such as when a <a data-cite="device-posture#device-posture-change">device's posture changes</a> or when the [=device change notification steps|set of available media devices changes=].</p>
```

(See https://respec.org/xref/ to look up linking syntax.) Respec might need you to add `mediacapture-streams` to adjust the `xref` or `data-cite` configuration to find this. See https://github.com/speced/respec/wiki/Auto-linking-external-references.

Reply:
Thanks for these links! 

I'm a little confused on some of the referencing (e.g. in the below change I don't understand why I needed to do `<a data-cite="html#Worker">Worker</a>` instead of just `{{Worker}}` when the enclosing tag has the data-cite `<p class="practicedesc" data-cite="html">`) but I got the desired output even if the redundancy doesn't make sense to me.

I've been force-pushing as I've been updating, but would it be better to just stack 3 or 4 fix commits on top of each other and then whoever merges it does a squash?

Reply:
It looks like this document is using a pretty old version of the Respec template, and it's missing `xref: "web-platform"` as recommended by https://respec.org/docs/#getting-started. Adding that lets you use {{Worker}} and {{Window}}.

You'll also see a warning in the top right that the '"respec-w3c-common" profile has been deprecated in favor of the "respec-w3c" profile'. https://respec.org/docs/#getting-started also has the new script tag, but you'll probably want to do that in a separate PR because it'll make you set the `group` for the document, and there may be other bits of the config that you can then delete. Feel free to ask me or mail the https://www.w3.org/community/speced-cg/ list for help.

When iterating on PRs, I generally stack several commits and expect people to squash-merge, so that people can see how I reacted to their comments. I'll rebase and force-push if upstream changes.

Reply:
Sorry about this - I've added the xref and corrected the citations.   Respec was upgraded in 4f1a3fbb0172247416bdd2d473909c40e7450cba